f5lbaasdriver/v2/bigip/driver_v2.py

```diff
@@ -125,6 +124,10 @@ def post_fork_callback(resources, event, trigger):
         post_fork_callback.__name__ += '_' + str(self.env)
         return post_fork_callback
 
+    def _handle_driver_error(self, context, loadbalancer,
```

Comment:
Hey, there is an issue that addresses this call in the driver_v2 module. I've come up with a reasonable explanation as to why the call was broken. Take a look: https://github.com/F5Networks/f5-openstack-lbaasv2-driver/issues/739

Not sure if we can (or should) call that function in neutron-lbaas.